curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641daabed8d0584b1150c953.md

```diff
@@ -34,13 +34,13 @@ assert.match(code, /<p\s*class=("|')bio\1>\$\{\s*bio\.length\s*>\s*50/)
 If the `bio` text is greater than `50` characters, you should extract the first 50 characters with `slice()` and replace the rest with `...`. Don't forget that indexes are zero-based.
 
 ```js
-assert.match(code, /<p\s*class=("|')bio\1>\$\{\s*bio\.length\s*>\s*50\s*\?\s*bio\.slice\(\s*0\,\s*50\s*\)\s*\+\s*("|')\.\.\.\2\s*\:/)
+assert.match(code, /<p\s*class=("|')bio\1>\$\{\s*bio\.length\s*>\s*50\s*\?\s*bio\.slice\(\s*0\,\s*50\s*\)\s*\+\s*("|')\.\.\.\2\s*\:|<p\s*class="bio">\$\{bio\.length\s*>\s*50\s*\?\s*`\$\{bio\.slice\(0,\s*50\)\}\.\.\.`\s*:\s*bio\}<\/p>/);
```

Comment:
Thank you for your contribution @iprime2  Can you try this code, this is the simplified version.
```suggestion
assert.match(code, /<p\s*class=("|')bio\1>\$\{\s*bio\.length\s*>\s*50\s*\?\s*(`\$\{bio\.slice\(0, 50\)\}\.\.\.`|bio\.slice\(0, 50\) \+ '...')\s*:\s*bio\}<\/p>/);
```

Reply:
let me test this if it works I will commit your suggestions.